app/models/client.rb

```diff
@@ -193,6 +193,12 @@ def clients_with_dupe_contact_info
     Client.after_consent.where(intake: matching_intakes).pluck(:id)
   end
 
+  def preferred_language
+    return intake.preferred_interview_language if intake.preferred_interview_language && intake.preferred_interview_language != "en"
```

Comment:
When I set a client language to "Spanish" I see Spanish in the `Language` column. But when I set a client to "English", it shows as "N/A" in the `Language` column. Is that expected?

<img width="856" alt="set-client-to-english" src="https://user-images.githubusercontent.com/9101728/111848688-b02dbb80-88d9-11eb-830c-d46c39c57c9c.png">

<img width="1192" alt="language-col-set-to-n:a" src="https://user-images.githubusercontent.com/9101728/111848691-b1f77f00-88d9-11eb-8b0c-6d8f3e3fa87e.png">

Reply:
@coltborg Ah, this was a bug where if preferred interview language was english, we wanted to check if the locale was spanish and use that as language, but then weren't falling back on locale if preferred language was nil (like for valet clients). I updated this and added a test. Thanks!